targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFab/Private/PlayFab.cpp.ejs

```diff
@@ -35,3 +35,4 @@ class FPlayFab : public IPlayFab
 IMPLEMENT_MODULE(FPlayFab, PlayFab)
 
 DEFINE_LOG_CATEGORY(LogPlayFab);
+DEFINE_LOG_CATEGORY(LogPlayFabTests);
```

Comment:
This is slightly confusing to me.
You are adding LogPlayFabTests here, but you're removing multiple uses of it up above.
Can you clarify that?